core-bundle/src/Resources/contao/dca/tl_user.php

```diff
@@ -180,7 +181,7 @@
 			'eval'                    => array('mandatory'=>true, 'tl_class'=>'w50'),
 			'options_callback' => static function ()
 			{
-				return System::getLanguages(true);
+				return System::getContainer()->get(Locales::class)->getDisplayNames(System::getContainer()->getParameter('contao.locales'), null, true);
```

Comment:
I think we should have a `Locales::getInstalledLocales()`. The backend locales are already passed to the service, but not used atm as far as I could see.

Reply:
but they _should_ use the service and not a parameter? Because the service would allow us to change the parameter etc. without affecting the API?

Reply:
We could add such a method, but is it really worth it?
Also I’m not sure about naming them “installed”, maybe `getTranslatedLocales()` or `getEnabledLocales()` or `getActiveLocales()`?

Reply:
As briefly discussed in Slack, we want to rename the parameter to `contao.intl.enabled_locales` and add a method `getEnabledLocales().